Source/Core/DolphinWX/Main.cpp

```diff
@@ -318,22 +318,22 @@ void DolphinApp::AfterInit()
 
 	if (!SConfig::GetInstance().m_analytics_permission_asked)
```

Comment:
I reckon you should just have some sort of global variable called like `enable_analytics_prompt` or something and have that check against line 319, and check against that boolean value instead of commenting out the message box.

Reply:
Changed the logic around so that it will call `m_analytics_will_prompt` from the dolphin config files.